libs/common/domain/src/lib/model/user/user.model.ts

```diff
@@ -0,0 +1,21 @@
+export interface UserModel {
+  id: string
+  profile: string
+  username: string
+  name: string
+  surname: string
+  email: string
+  organisation: string
+  profileIcon?: string
+}
+
+export interface UserModela {
+  enabled?: boolean
+  emailAddresses?: Set<string>
+  organisation?: string
+  kind?: string
+  lastLoginDate?: string
+  accountNonExpired?: boolean
+  accountNonLocked?: boolean
+  credentialsNonExpired?: boolean
+}
```

Comment:
Modela? Por favor?

Reply:
was to remove the dependence on the model :laughing: